applications/DEMApplication/tests/test_post_process.py

```diff
@@ -46,11 +44,8 @@ def test_gid_printing_many_results(self):
         model = KratosMultiphysics.Model()
         auxiliary_functions_for_tests.CreateAndRunStageInSelectedNumberOfOpenMPThreads(TestPostProcessClass1, model, parameters_file_name, 1)
 
-    def tearDown(self):
-        file_to_remove = os.path.join("post_process_tests_files", "TimesPartialRelease")
-        kratos_utils.DeleteFileIfExisting(GetFilePath(file_to_remove))
-        file_to_remove = os.path.join("post_process_tests_files", "flux_data_new.hdf5")
-        kratos_utils.DeleteFileIfExisting(GetFilePath(file_to_remove))
+    @staticmethod
```

Comment:
Dangerous. Are we sure that putting this decorator does not spoil the override of the function??